mosaic-runtime/src/commonTest/kotlin/com/jakewharton/mosaic/ui/RowColumnTest.kt

```diff
@@ -0,0 +1,2399 @@
+			IntOffset(
+				0,
+				(extraSpace / 2) + size * 2,
+			),
+		)
+	}
+
+	@Test fun testColumn_withSpaceEvenlyArrangement() {
+		val size = 50
+
+		val rootNode = mosaicNodesWithMeasureAndPlace {
+			Center {
+				Column(
+					modifier = Modifier.fillMaxHeight(),
+					verticalArrangement = Arrangement.SpaceEvenly,
+				) {
+					Container(width = size, height = size)
+					Container(width = size, height = size)
+					Container(width = size, height = size)
+				}
+			}
+		}
+
+		val gap = (rootNode.height - size * 3) / 4
+
+		val firstChildContainerNode = rootNode.children[0].children[0].children[0]
+		val secondChildContainerNode = rootNode.children[0].children[0].children[1]
+		val thirdChildContainerNode = rootNode.children[0].children[0].children[2]
+
+		assertThat(firstChildContainerNode.position).isEqualTo(IntOffset(0, gap))
+		assertThat(secondChildContainerNode.position).isEqualTo(IntOffset(0, size + gap * 2))
+		assertThat(thirdChildContainerNode.position).isEqualTo(IntOffset(0, size * 2 + gap * 3))
+	}
+
+	@Test fun testColumn_withSpaceBetweenArrangement() {
+		val size = 50
+
+		val rootNode = mosaicNodesWithMeasureAndPlace {
+			Center {
+				Column(
+					modifier = Modifier.fillMaxHeight(),
+					verticalArrangement = Arrangement.SpaceBetween,
+				) {
+					Container(width = size, height = size)
+					Container(width = size, height = size)
+					Container(width = size, height = size)
+				}
+			}
+		}
+
+		val gap = (rootNode.height - size * 3) / 2
+
+		val firstChildContainerNode = rootNode.children[0].children[0].children[0]
+		val secondChildContainerNode = rootNode.children[0].children[0].children[1]
+		val thirdChildContainerNode = rootNode.children[0].children[0].children[2]
+
+		assertThat(firstChildContainerNode.position).isEqualTo(IntOffset.Zero)
+		assertThat(secondChildContainerNode.position).isEqualTo(IntOffset(0, size + gap))
+		assertThat(thirdChildContainerNode.position).isEqualTo(IntOffset(0, size * 2 + gap * 2))
+	}
+
+	@Test fun testColumn_withSpaceAroundArrangement() {
+		val size = 50
+
+		val rootNode = mosaicNodesWithMeasureAndPlace {
+			Center {
+				Column(
+					modifier = Modifier.fillMaxHeight(),
+					verticalArrangement = Arrangement.SpaceAround,
+				) {
+					Container(width = size, height = size)
+					Container(width = size, height = size)
+					Container(width = size, height = size)
+				}
+			}
+		}
+
+		val gap = (rootNode.height - size * 3) / 3
+
+		val firstChildContainerNode = rootNode.children[0].children[0].children[0]
+		val secondChildContainerNode = rootNode.children[0].children[0].children[1]
+		val thirdChildContainerNode = rootNode.children[0].children[0].children[2]
+
+		assertThat(firstChildContainerNode.position).isEqualTo(IntOffset(0, gap / 2))
+		assertThat(secondChildContainerNode.position).isEqualTo(IntOffset(0, size + (gap * 3 / 2)))
+		assertThat(thirdChildContainerNode.position).isEqualTo(IntOffset(0, size * 2 + (gap * 5 / 2)))
```

Comment:
I'm not sure whether it makes sense to do math in all these tests or to put hard coded values here (since the input values are hard coded). Since you did them all with math we can leave them for now, but I _think_ I would prefer hard coded values in the future.